content/docs/overview/tracers.md

```diff
@@ -1,3 +1,51 @@
+
+### Starting a new Trace
+Finally, a new trace can be started by creating a new span. A span must specify the operation name. Operation names are used for providing semantic meaning to the trace. It is important to recognize whether we need to start a new trace or connect with an already-active trace. If there's an active trace, you'll start a span with the active span as its parent. Otherwise, the span you start will have no parent.OpenTracing supports two kinds of direct causal relationships:
+
+1. **ChildOf**: The parent span is dependent on the child span for its execution.
+2. **FollowsFrom**: This relationship is used to model asynchronous executions. The parent span is fully independent of the child span.
+
+
+### Accessing the Active Span
+The GlobalTracer can be used to access the currently ActiveSpan and log events. Further, ActiveSpans can be accessed through a scopeManager in some languages. Refer to the specific language guide for more implementation details.
+
+### Propagating a Trace with Inject/Extract
+In order to trace across process boundaries in distributed systems, services need to be able to continue the trace injected by the client that sent each request. OpenTracing allows this to happen by providing inject and extract methods that encode a span's context into a carrier.
+The inject method allows for the SpanContext to be passed on to a carrier. For example, passing the trace information into the client's request so that the server you send it to can continue the trace. The extract method does the exact opposite. It extract the SpanContext from the carrier. For example, if there was an active request on the client side, the developer must extract the span context using the `io.opentracing.Tracer.extract` method.
+
+## Tracing Systems
+
+The following tracing backends have OpenTracing-compatible tracing libraries:
+
+Tracing System | Supported Languages
+------------ | -------------
+CNCF Jaeger | [Java](https://github.com/jaegertracing/jaeger-client-java), [Go](https://github.com/jaegertracing/jaeger-client-go), [Python](https://github.com/jaegertracing/jaeger-client-python), [Node.js](https://github.com/jaegertracing/jaeger-client-node), [C++](https://github.com/jaegertracing/cpp-client), [C#](https://github.com/jaegertracing/jaeger-client-csharp)
+Appdash | [Go](https://github.com/sourcegraph/appdash)
+LightStep | [Go](https://github.com/lightstep/lightstep-tracer-go), [Python](https://github.com/lightstep/lightstep-tracer-python), [Javascript](https://github.com/lightstep/lightstep-tracer-javascript), [Objective-C](https://github.com/lightstep/lightstep-tracer-objc), [Java](https://github.com/lightstep/lightstep-tracer-java), [PHP](https://github.com/lightstep/lightstep-tracer-php), [Ruby](https://github.com/lightstep/lightstep-tracer-ruby), [C++](https://github.com/lightstep/lightstep-tracer-cpp)
+Hawkular | [Java](https://github.com/hawkular/hawkular-apm)
```

Comment:
Remove this one. We (Hawkular team) joined the Jaeger team more than a year ago :-)

Reply:
Yes, please!